bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/ShadeThingHandler.java

```diff
@@ -0,0 +1,339 @@
+                                (UpDownType.UP == command) && !Objects.requireNonNull(capabilities).isPrimaryInverted()
+                                        ? PercentType.HUNDRED
+                                        : PercentType.ZERO);
+                        webTargets.moveShade(shadeId, new Shade().setShadePosition(position));
+                        break;
+                    } else if (StopMoveType.STOP == command) {
+                        webTargets.stopShade(shadeId);
+                        break;
+                    }
+                    throw new IllegalArgumentException(INVALID_COMMAND);
+
+                case CHANNEL_SHADE_SECONDARY_POSITION:
+                    if (command instanceof PercentType) {
+                        position.setPosition(SECONDARY_POSITION, ((PercentType) command));
+                        webTargets.moveShade(shadeId, new Shade().setShadePosition(position));
+                        break;
+                    } else if (command instanceof UpDownType) {
+                        position.setPosition(SECONDARY_POSITION,
+                                (UpDownType.UP == command)
+                                        && !Objects.requireNonNull(capabilities).supportsSecondaryOverlapped()
+                                                ? PercentType.ZERO
+                                                : PercentType.HUNDRED);
+                        webTargets.moveShade(shadeId, new Shade().setShadePosition(position));
+                        break;
+                    } else if (StopMoveType.STOP == command) {
+                        webTargets.stopShade(shadeId);
+                        break;
+                    }
+                    throw new IllegalArgumentException(INVALID_COMMAND);
+
+                case CHANNEL_SHADE_VANE:
+                    if (command instanceof PercentType) {
+                        position.setPosition(VANE_TILT_POSITION, ((PercentType) command));
+                        webTargets.moveShade(shadeId, new Shade().setShadePosition(position));
+                        break;
+                    } else if (command instanceof UpDownType) {
+                        position.setPosition(VANE_TILT_POSITION,
+                                UpDownType.UP == command ? PercentType.HUNDRED : PercentType.ZERO);
+                        webTargets.moveShade(shadeId, new Shade().setShadePosition(position));
+                        break;
+                    }
+                    throw new IllegalArgumentException(INVALID_COMMAND);
+
+                case CHANNEL_SHADE_COMMAND:
+                    if (command instanceof StringType) {
+                        if (COMMAND_IDENTIFY.equals(((StringType) command).toString())) {
+                            webTargets.jogShade(shadeId);
+                            break;
+                        } else if (COMMAND_CALIBRATE.equals(((StringType) command).toString())) {
```

Comment:
The README says:
- On Generation 1/2 hubs, the valid values for the `command` channel are: `CALIBRATE`, `IDENTIFY`.
On Generation 3 gateways, the only valid value is: `IDENTIFY`.